content/efcore/getting-started/samples/defining-unique-identifiers/primary-key-fluent-api/README.md

```diff
@@ -0,0 +1,8 @@
+# Primary Key Using Fluent API
+
+This code example demonstrates how to use EF Core to set a property as the primary key using Fluent API.
```

Comment:
This code example demonstrates how to set an unconventionally-named property as the primary key in EF Core via Fluent API.